azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/resources/templates.json

```diff
@@ -600,6 +600,214 @@
       }
+    },
+    {
+      "id": "AssistantSkillTrigger-Java",
+      "metadata": {
+        "name": "AssistantSkillTrigger",
+        "description": "$AssistantSkillTrigger_description",
+        "defaultFunctionName": "assistantSkillTriggerJava",
+        "language": "Java"
+      },
+      "bundle": ["~4"],
+      "function": {
+        "disabled": false,
+        "bindings": [
+          {
+            "type": "assistantSkillTrigger",
+            "name": "taskDescription",
+            "direction": "in",
+            "functionDescription": "Create a new todo task",
+            "parameterDescriptionJson": "{}",
+            "model": "gpt-3.5-turbo"
+          }
+        ]
+      },
+      "files": {
+        "function.java": "package $packageName$;\n\nimport com.microsoft.azure.functions.annotation.*;\nimport com.microsoft.azure.functions.*;\n\nimport java.util.*;\nimport java.util.concurrent.CopyOnWriteArrayList;\n\npublic class AddTodoFunction {\n\n    // In-memory list to store todo items\n    private static final List<TodoItem> todoList = new CopyOnWriteArrayList<>();\n\n    @FunctionName(\"AddTodo\")\n    public void addTodo(\n        @AssistantSkillTrigger(\n                name = \"$name$\",\n                functionDescription = \"$functionDescription$\",\n                parameterDescriptionJson = \"$parameterDescriptionJson$\",\n                model = \"$model$\"\n        ) String taskDescription,\n        final ExecutionContext context) {\n\n        if (taskDescription == null || taskDescription.isEmpty()) {\n            throw new IllegalArgumentException(\"Task description cannot be empty\");\n        }\n\n        context.getLogger().info(\"Adding todo: \" + taskDescription);\n\n        // Generate a random ID for the todo item\n        String todoId = UUID.randomUUID().toString().substring(0, 6);\n\n        // Create a new TodoItem and add it to the in-memory list\n        TodoItem todoItem = new TodoItem(todoId, taskDescription);\n        todoList.add(todoItem);\n\n        context.getLogger().info(\"Todo added with ID: \" + todoId);\n    }\n\n    // Simple TodoItem class\n    static class TodoItem {\n        private String id;\n        private String description;\n\n        public TodoItem(String id, String description) {\n            this.id = id;\n            this.description = description;\n        }\n\n        public String getId() {\n            return id;\n        }\n\n        public String getDescription() {\n            return description;\n        }\n    }\n}\n"
```

Comment:
please change the class name to `$className$` to accept user's input

Reply:
please also add some comments to tell users which library they need to add to use the ai templates

Reply:
please add the missing import statement for AssistantSkillTrigger
```
import com.microsoft.azure.functions.openai.annotation.assistant.AssistantSkillTrigger;
```

Reply:
For the parameters of annotation `AssistantSkillTrigger`, if you want to make users could set the value during `azure-functions:add`, please update the `settings` part of `assistantSkillTrigger` in `bindings.json`, and add it to the userPrompt in metadata

Please use same name in `settings` and userPrompt, so that functions maven plugin will get the value from user and replace $property_name$ in template with user's input

Here is the reference of sql,
```
{
      "type": "sql",
      "displayName": "$sql_input_displayName",
      "direction": "in",
      "enabledInTryMode": false,
      "settings": [
        {
          "name": "object",
          "value": "string",
          "defaultValue": "[dbo].[table1]",
          "required": true,
          "label": "$sql_input_object_label",
          "help": "$sql_input_object_help"
        },
        {
          "name": "SqlConnectionString",
          "value": "string",
          "defaultValue": "",
          "required": true,
          "resource": "Sql",
          "label": "$sql_input_connection_label",
          "help": "$sql_input_connection_help"
        }
      ]
    },
```

if is not required, you may set a default value or remove it.

Reply:
thanks Hanxiao, i updated. I think we can go default atm for these parameters